pkg/e2e/framework/ssh.go

```diff
@@ -0,0 +1,54 @@
+package framework
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net"
+	"os"
+
+	"golang.org/x/crypto/ssh"
+)
+
+func createSSHClient(sshConfig *SSHConfig) (*ssh.Client, error) {
+	signer, err := LoadPrivateKey(sshConfig.Key)
+	if err != nil {
+		return nil, err
+	}
+
+	config := &ssh.ClientConfig{
+		User: sshConfig.User,
+		Auth: []ssh.AuthMethod{
+			ssh.PublicKeys(signer),
+		},
+		HostKeyCallback: func(hostname string, remote net.Addr, key ssh.PublicKey) error {
+			return nil
+		},
+	}
+
+	client, err := ssh.Dial("tcp", sshConfig.Host+":22", config)
+	if err != nil {
+		return nil, fmt.Errorf("failed to dial: " + err.Error())
+	}
+	return client, nil
+
+}
+
+func LoadPrivateKey(prkey string) (ssh.Signer, error) {
+	fp, err := os.Open(prkey)
+	if err != nil {
+		return nil, fmt.Errorf("unable to open id_rsa file: %v", err)
+	}
+	defer fp.Close()
+
+	buf, err := ioutil.ReadAll(fp)
+	if err != nil {
+		return nil, fmt.Errorf("unable to read id_rsa file: %v", err)
```

Comment:
This should be: `Errorf("unable to open %q: %v", prkey, err)`.

Reply:
Fixed